src/nvim/api/ui.c

```diff
@@ -611,6 +614,38 @@ static void remote_ui_event(UI *ui, char *name, Array args, bool *args_consumed)
     }
   }
 
+  if (ui->ui_ext[kUIWildmenu]) {
```

Comment:
brief comment would be helpful, e.g.

    // Translate popupmenu_xx to legacy popupmenu_xx. 
    if (ui->ui_ext[kUIWildmenu]) {